app/assets/stylesheets/views/_detailed-guide.scss

```diff
@@ -19,4 +19,14 @@
       margin: 0 0 $gutter-half $gutter-half;
     }
   }
+
+  // To allow .back-to-content to settle in at the bottom after scrolling past.
+  .sidebar-with-body {
+    position: relative;
+  }
+
+  // To compensate for the -15px from @include sidebar-with-body.
```

Comment:
`-$gutter` is *eh*, most everyone working in this codebase knows how big $gutter is anyway (except for me, obviously, since I forgot to use it).